### PR TITLE
feat:  added additional notes to aws-instructions

### PR DESCRIPTION
**Pull request by pakawaka1**
I added some additional notes for potential future issues.

### CODE REVIEW

File: instructions.md [hunk 1]
```diff
@@ -50,13 +51,13 @@ The first step will be uploading the images into a docker image repository onto
 
 - Open your docker terminal if you are on Windows. I believe Mac's have the docker machine initialized automatically so you can go to a normal terminal.
 Run the aws cli command listed on the push instructions:
-`aws ecr get-login --no-include-email --region us-west-2`
+`acd`
```
Comment (Fleury14) [inline instructions.md]: Is acd a shortcut for that command?

File: instructions.md [hunk 2]
```diff
@@ -208,33 +209,34 @@ You can also use `eb local status` in a seperate terminal to check the status of
 
 ### Step 5: Make sure your EB role has access to the repository
 
-To do this, you will need to search services for *IAM*. Under IAM resources click on *Roles: #* and find `aws-elasticbeanstalk-ec2-role` and click on it. Under the permissions tab, you will need to click the Attack policy and apply the `AmazonEC2ContainerRegistryReadOnly` policy. This gives EB access to the images you just pushed.
+To do this, you will need to search services for *IAM*. Under IAM resources click on *Roles: #* and find `aws-elasticbeanstalk-ec2-role` and click on it. Under the permissions tab, you will need to click the `Attach` button and apply the `AmazonEC2ContainerRegistryReadOnly` policy. This gives EB access to the images you just pushed.
 
 ### Step 6: git add/commit your Dockerrun.aws.json file
-
+e
```
Comment (Fleury14) [inline instructions.md]: Typo

File: instructions.md (hunk 1 above)
Comment (pakawaka1) [reply instructions.md]: no, it's a typo

File: instructions.md [hunk 3]
```diff
@@ -50,13 +51,12 @@ The first step will be uploading the images into a docker image repository onto
 
 - Open your docker terminal if you are on Windows. I believe Mac's have the docker machine initialized automatically so you can go to a normal terminal.
 Run the aws cli command listed on the push instructions:
```
Comment (Fleury14) [inline instructions.md]: You may still want to leave the old `aws ecr-login` command

**Comment by pakawaka1** [comment]
Just updated.  Fixed typos.  Please take a look and merge.

**Comment by Fleury14** [comment]
@pakawaka1 The only change left I would make would be to make sure the aws ecr login is in the instructions and not deleted.

File: instructions.md (hunk 3 above)
Comment (pakawaka1) [reply instructions.md]: Didn't know I deleted it.

Comment (Fleury14) [reply instructions.md]: Thats why we review. I've totally done that before. 👍

Comment (pakawaka1) [reply instructions.md]: fixed it.  thanks for catching that

**Review by Fleury14 — APPROVED** [review]
Looks good. I will merge it.

**Pull request merged**